<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JrnWorkReports</class>
 <widget class="QDialog" name="JrnWorkReports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/icons/icons/reports.png</normaloff>:/icons/icons/reports.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Поиск</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>№ заказа:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="eSearchNumber">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Мастер:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cSearchEmployee">
        <property name="minimumSize">
         <size>
          <width>181</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>231</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>222</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="bClear">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/icons/filter_clear.png</normaloff>:/icons/icons/filter_clear.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="bAdd">
       <property name="text">
        <string>Новый</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/report_add.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bView">
       <property name="text">
        <string>Просмотр</string>
       </property>
       <property name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/icons/icons/report_magnify.png</normaloff>:/icons/icons/report_magnify.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bEdit">
       <property name="text">
        <string>Изменить</string>
       </property>
       <property name="icon">
        <iconset resource="Icons.qrc">
         <normaloff>:/icons/icons/report_edit.png</normaloff>:/icons/icons/report_edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bDelete">
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/report_delete.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/door_out.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tview">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
